backend/messaging.py

```diff
@@ -71,27 +95,72 @@ def message_create_get(request, room_id):
                     result_page = paginator.paginate_queryset(messages, request)
                     return paginator.get_paginated_response(result_page)
             else:
+
+                # Else, return a message telling the user that the room doesn't exist.
                 return Response(data="No such room", status=status.HTTP_404_NOT_FOUND)
         else:
+
+            # Else, return a bad request message
             return Response(
                 params_serializer.errors, status=status.HTTP_400_BAD_REQUEST
             )
 
-    elif request.method == "POST":
+
+    def post(self, request, room_id, org_id):
+        """
+        Create a message in a specified room of a specified organization.
+
+        Args:
+            request (dict): The request body
+            room_id (str): The id of the room where the request is being made
+            org_id (str): The id of the organization of the user
+
+        Returns:
+            201: {
+                    "status": response["message"],
+                    "event": "message_create",
+                    "message_id": "message_id",
+                    "room_id": room_id,
+                    "thread": False,
+                    "data": {
+                        "sender_id": data["sender_id"],
+                        "message": data["message"],
+                        "created_at": data["created_at"],
+                    },
+                }
+        
+         Raises:
+            424: failed dependency.
+            404: Sender not in room.
+            404: Room does not exist.
+            400: Bad Request.
+        """
+        # specify the organizaiton id
+        data_storage = DataStorage()
+        data_storage.organization_id = org_id
+
+        # add the room_id to the request data
         request.data["room_id"] = room_id
+        print(request)
+
+        # serialize the data
         serializer = MessageSerializer(data=request.data)
 
+        # check if the serialized data is valid
         if serializer.is_valid():
+            # if true continue
             data = serializer.data
             room_id = data["room_id"]  # room id gotten from client request
 
+            # Check to see if the room exists in the database
             room = DB.read_query("dm_rooms", query={"_id": room_id})
```

Comment:
U already instantiated the DataStorage class as data_storage, u dont need to be using DB.read_query instead data_stotage.read_query

Reply:
@paisoncodes. Or you can also just remove the data_storage you instantiated and use what @man-of-mind suggested above.